src/Browser/Test/HasBrowser.php

```diff
@@ -95,6 +96,7 @@ protected function browser(array $options = [], array $server = []): KernelBrows
 
         $browserOptions = [
             'source_dir' => $_SERVER['BROWSER_SOURCE_DIR'] ?? './var/browser/source',
+            'source_debug' => $_SERVER['BROWSER_SOURCE_DEBUG'] ?? false,
```

Comment:
Can you add this option to https://github.com/zenstruck/browser#configuration?

Reply:
Added in https://github.com/zenstruck/browser/commit/a99150a8af3adb9a4f7113c94dd56ae4441d63d9.